react-hot-button-component/src/index.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class Button extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { clicked: 0, color: 'white' };
```

Comment:
We always want to keep our state as simple as possible so here we only really need to keep track of the number of clicks in state. So remove `color` from state. The color can be calculated from the `clicked` property in state so there is no point in trying to keep track of it in state.